.github/ui-e2e-tests/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@traceable/e2e",
+  "name": "@hypertrace/e2e",
   "version": "0.0.0",
-  "description": "E2E Tests for Traceable UI App",
+  "description": "E2E Tests for hypertrace UI App",
   "author": "",
   "license": "MIT",
   "scripts": {
     "tsc": "tsc",
-    "install-web-driver": "webdriver-manager clean && webdriver-manager update",
+    "install-web-driver": "webdriver-manager clean && webdriver-manager update --gecko false",
```

Comment:
is this the problem `gecko` on local mac as well?

Reply:
yup. It wasn't prominent earlier. But I observed it yesterday. And we anyways use chrome for test so it's okay to disable it.